docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
-      </Text>
-    </Link>
+      </Link>
+    </Text>
   }
+  text="There was a problem converting to a personal account."
 >
-  <Icon
-    accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+  <Button size="lg" disabled text="Convert to personal account" />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`
+When using Tooltip with [IconButton](/IconButton), avoid repetitive labeling. The accessibilityLabel provided to IconButton should describe the intent of the button, not the icon itself. For instance, use “Settings” instead of “Cog icon”. Tooltip text can expand upon that intention, as seen with the cog IconButton. If the Tooltip text and IconButton accessibilityLabel contain the same content, pass an empty string to \`accessibilityLabel\`, as seen with the send IconButton.`}
+  >
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Tooltip text="Customize performance stats for your paid ads">
+  <IconButton
+    accessibilityLabel="Settings"
+    bgColor="white"
+    icon="cog"
+    iconColor="darkGray"
+    size="lg"
+  />
+</Tooltip>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Tooltip text="Send Pin">
+  <IconButton
+    accessibilityLabel=""
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
   />
 </Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Disabled elements"
+    description={`
+Tooltips must be paired with an interactive, focusable element, like [Button](/Button) or [IconButton](/IconButton). They cannot be paired with anything disabled, because this prevents keyboard users from triggering Tooltip and consuming its content. To test if you’re using a Tooltip properly, use your keyboard rather than your mouse to trigger Tooltip.
+
+If you need to explain why an item is disabled, consider adding plain [Text](/Text) near the disabled item, or an \`info-circle\` [IconButton](/IconButton) in a different part of the component.
 `}
   />,
 );
 
 card(
-  <Example
-    id="inline"
-    name="Inlining Tooltips"
-    description="Pass this flag in when rendering the tooltip around an inline element such as an Icon,
-      IconButton, or a Button with the inline prop set. This will help correctly position the tooltip
-      alongside the element."
-    defaultCode={`
-<Box>
-  <Box padding={2}>
-    <Tooltip inline text="Inline tooltip">
-      <Button
-        accessibilityLabel="This tooltip wraps an inline button and occupies the same space"
-        color="gray"
-        text="Inline"
+  <MainSection
+    name="Localization"
+    description={`If your application requires localization, be sure to localize the \`text\` and \`accessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
```

Comment:
I thought we were ditching the "If your application requires localization" phrase?

Reply:
DONE